common/src/main/java/com/duckduckgo/app/global/formatters/time/TimeDiffFormatter.kt

```diff
@@ -117,6 +117,7 @@ class RealTimeDiffFormatter @Inject constructor(private val context: Context) :
 
     companion object {
         private val FORMATTER_DAYS = SimpleDateFormat("yyyy-MM-dd")
+        private const val PATTERN_FORMATTER_DAYS = "yyyy-MM-dd"
```

Comment:
Kept only the pattern here as reusing the `SimpleDateFormat` was crashing the app.